### PR TITLE
feat: added inventory table shopify implementation

### LINKED ISSUE #7488: [Shopify] Implement the Inventory table
MindsDB integrates with Shopify, so you can access your Shopify data and make relevant forecasts with MindsDB.

## Resources

Familiarize yourself with the existing Shopify integration: https://github.com/mindsdb/mindsdb/tree/staging/mindsdb/integrations/handlers/shopify_handler

## Steps to Follow

Currently, the Shopify integration with MindsDB accesses tables from Shopify API that store products and customers.

The task is to extend [this implementation](https://github.com/mindsdb/mindsdb/tree/staging/mindsdb/integrations/handlers/shopify_handler) to include the Inventory table from Shopify API.

**Please note that we are not assigning this type of issue.
The first PR that solves this will be accepted** :fireworks:

## :loudspeaker: Additional rewards :trophy:

Each integration test brings 15+ points for SWAG and entry into the draw for a :point_down: :
-  :computer:  [Razer Blade 16 Laptop](https://www.razer.com/gaming-laptops/Razer-Blade-16/RZ09-0483SEJ3-R3U1)
- :shirt: :bear:  [MindsDB Swag](https://store.covver.io/mindsdb/collections/swag-collection)

:information_source: For more info check out https://mindsdb.com/hacktoberfest/ :point_left: 

**Please note that we are not assigning this type of issue.
The first PR that solves this will be accepted** :fireworks:

### PR DESCRIPTION
**Pull request by sahil9001**
## Description

Please include a summary of the change and the issue it solves. 
This PR implements the `Inventory` table from shopify API

Fixes #7488 

## Type of change

(Please delete options that are not relevant)

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [x] ⚡ New feature (non-breaking change which adds functionality)
- [ ] 📢 Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [ ] 📄 This change requires a documentation update

## Verification Process

To ensure the changes are working as expected:

 - [ ]   Test Location: Specify the URL or path for testing.
 - [ ]   Verification Steps: Outline the steps or queries needed to validate the change. Include any data, configurations, or actions required to reproduce or see the new functionality.

## Additional Media:

- [ ] I have attached a brief loom video or screenshots showcasing the new functionality or change.

## Checklist:

- [ ] My code follows the style guidelines(PEP 8) of MindsDB.
- [ ] I have appropriately commented on my code, especially in complex areas.
- [ ] Necessary documentation updates are either made or tracked in issues.
- [ ] Relevant unit and integration tests are updated or added.

### CODE REVIEW

**Comment by ZoranPandovski** [comment]
recheckcla

**Comment by mindsdbadmin** [comment]
**CLA Assistant Lite bot:** <br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/mindsdb/mindsdb_native/blob/stable/assets/contributions-agreement/individual-contributor.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.<br/>
   - - -
   I have read the CLA Document and I hereby sign the CLA
   - - -
   <sub>You can retrigger this bot by commenting **recheck** in this Pull Request</sub>

**Comment by martyna-mindsdb** [comment]
This has been implemented. You can look at other hacktoberfest issues.

**Comment by sahil9001** [comment]
Well @martyna-mindsdb , is it fair to close the first PR (which was mine) and accept the second one? I think I was not given chance to even learn about this and other person's PR got accepted much before I can even comprehend. Anyways, no issues with it, just a thing from which I feel discouraged to make PRs on this repo.

File: mindsdb/integrations/handlers/shopify_handler/shopify_tables.py [hunk 1]
```diff
@@ -198,4 +198,53 @@ def get_orders(self, **kwargs) -> List[Dict]:
         api_session = self.handler.connect()
         shopify.ShopifyResource.activate_session(api_session)
         orders = shopify.Order.find(**kwargs)
-        return [order.to_dict() for order in orders]
\ No newline at end of file
+        return [order.to_dict() for order in orders]
+
+class InventoryItemsTable(APITable):
+    """The Shopify Inventory Table implementation"""
```
Comment (MinuraPunchihewa) [inline mindsdb/integrations/handlers/shopify_handler/shopify_tables.py:204]: Shopify Inventory Items Table will make this more clear.

**Review by MinuraPunchihewa — CHANGES_REQUESTED** [review]

**Comment by martyna-mindsdb** [comment]
@sahil9001 
Please address the review comments above and provide screenshots of successful tests.

**Comment by ZoranPandovski** [comment]
Solved by https://github.com/mindsdb/mindsdb/pull/7494

**Pull request closed without merging**